ivy/container/elementwise.py

```diff
@@ -1893,9 +1893,9 @@ def static_cosh(
         out: Optional[ivy.Container] = None,
     ) -> ivy.Container:
         """
-        ivy.Container static method variant of ivy.cosh.
-        This method simply wraps the function, and so the docstring for
-        ivy.cosh also applies to this method with minimal changes.
+        ivy.Container instance method variant of ivy.cosh. This method simply wraps the
```

Comment:
This is the definition of a static method (view the decorator above the function). Hence, please mention this as a static method rather than instance.